rocketpy/mathutils/function.py

```diff
@@ -1112,6 +1112,9 @@ def low_pass_filter(self, alpha):
                 alpha * self.source[i] + (1 - alpha) * filtered_signal[i - 1]
             )
 
+        # Save the new csv file with filtered data
+        np.savetxt("filtered_data.csv", filtered_signal, delimiter=",")
```

Comment:
Good feature. One small note is that it might not be the user's will to always save the file and in this location.

Therefore, what I suggest is to add an optional parameter to the method `filepath` (or another name you prefer) that defaults to `None` (in this case nothing will be saved), but may receive the filepath (and filename) string that will go into the `np.savetxt`.

The final step would be describing this behavior on this parameter docstring.

As always, we are open to any other suggestions you might have.

Reply:
Good idea, thank you! I didn't thought about it, I will try to implement it!